requirements.txt

```diff
@@ -56,7 +54,7 @@ pyquery>=1.4.3
 python-dateutil>=2.8.2
 types-python-dateutil>=2.8.2
 python-magic==0.4.18    # Versions beyond the yanked .19 and .20 introduce form failures
-pymemcache>=4.0.0  # for django.core.cache.backends.memcached.PyMemcacheCache 
+pymemcache>=4.0.0  # for django.core.cache.backends.memcached.PyMemcacheCache
```

Comment:
nit - try not to bring in unrelated whitespace changes - put those in their own PR. (Otherwise time gets wasted later chasing back through revisions to find why the lines really changed.